docs/theming.md

```diff
@@ -23,3 +23,53 @@ To define a theme for Riot:
 
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
+
+Custom Themes
+=============
+Themes derived from the built in themes may also be defined in settings,
```

Comment:
should also probably describe how these things are named for use in the `default_theme` variable.

Reply:
Hmm, I *think* I know how this works although not 100% sure: @bwindels may have to weigh in here

Reply:
I think @turt2live means explaining that the internal name for a custom theme is `custom-${theme.name}`. I'll add some lines :+1: